apiserver/tools.go

```diff
@@ -110,18 +110,18 @@ func (h *toolsDownloadHandler) processGet(r *http.Request, st *state.State) ([]b
 	}
 	storage, err := st.ToolsStorage()
 	if err != nil {
-		return nil, errors.Annotate(err, "error getting tools storage")
+		return nil, errors.Annotate(err, "error getting agent binaries storage")
```

Comment:
error getting storage for agent binaries

(the old one reads like they were actually trying to say tool's storage, or something more like a possessive)
agent binaries' storage
?